website/docs/releases/2025/v2025.next.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Release xxxx.x
+slug: "/releases/xxxx.x"
+---
+
+:::::note
+xxxx.x has not been released yet! We're publishing these release notes as a preview of what's to come, and for our awesome beta testers trying out release candidates.
+
+To try out the release candidate, replace your Docker image tag with the latest release candidate number, such as xxxx.x.0-rc1. You can find the latest one in [the latest releases on GitHub](https://github.com/goauthentik/authentik/releases). If you don't find any, it means we haven't released one yet.
+:::::
+
+## Breaking changes
+
+- **Docker image deprecation notice for `beryju/authentik` and `beryju/authentik-*`**
+
+    The `beryju/authentik` and `beryju/authentik-*` Docker images are no longer being updated. Users are now encouraged to use the following images:
```

Comment:
Can you add some information about what these images are and how they are used? Are there only certain people impacted by this change, or does everyone [unwittingly] use these packages?